src/hadoop-node-manager/deploy/hadoop-node-manager.yaml.template

```diff
@@ -28,6 +28,13 @@ spec:
       labels:
         app: hadoop-node-manager
     spec:
+      hostAliases:
```

Comment:
hostAliases is only valid for non-hostNetwork?
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/

Reply:
No, it works on hostnetwork. And I found this logic in their code.
See the link: https://github.com/kubernetes/kubernetes/blob/197fc67693c2391dcbc652fc185ba85b5ef82a8e/pkg/kubelet/kubelet_pods.go#L298

Reply:
Get, seems doc doesn't update. This feature has been enabled since 1.8